PresentScreenings/Controllers/FilmRatingDialogController.cs

```diff
@@ -340,6 +333,18 @@ public void SetFilmRatingDialogButtonStates()
             _downloadFilmInfoButton.Enabled = OneOrMoreFilmsSelected();
         }
 
+        public void SelectFilms(List<Film> films)
+        {
+            NSMutableIndexSet rows = new NSMutableIndexSet();
+            foreach (var film in films)
+            {
+                int row = _filmTableDataSource.Films.IndexOf(film);
+                rows.Add(new NSIndexSet(row));
+            }
+            _filmRatingTableView.SelectRows(rows, false);
+            _filmRatingTableView.ScrollRowToVisible((nint)rows.First());
+        }
+
```

Comment:
Please consider to make this a private method, as to prevent calling it unthoughtful.
This probably means to use an event instead of an action, just like the Combine Titles and Uncombine Title popups.

Reply:
I considered, but I think the nature of the Download Film Info Controller is more elegant than the other two mentioned controllers, creating it's own buttons and actions.
The public method SelectFilms() can't easily be abused I think, because it always needs an existing Film Rating Dialog Controller and above that, setting a selection of films doesn't look that dangerous to me.
Sp, if it is wasn't for the event arguments, I would rather change the two using events as to create their own buttons and Actions.
Thanks for suggesting this, it made me rethink a lot about events and actions, but this time I won't follow up.